NativeCode/AudioPluginUtil.cpp

```diff
@@ -26,7 +30,7 @@ template<typename T>
 static void FFTProcess(UnityComplexNumber* data, int numsamples, bool forward)
 {
     unsigned int count = 1, numbits = 0;
-    while (count < numsamples)
+    while (count < (unsigned)numsamples)
```

Comment:
I would make the locals ints here instead of potentially forcing `numsamples` to overflow

Reply:
Okay, I'll resolve it with casting to int.

```suggestion
    while ((int)count < numsamples)
```

Reply:
Okay, I've fixed this casting.